Installation.md

```diff
@@ -66,7 +67,13 @@ haxelib install hlsdl
 haxelib install hldx
 ```
 
+You can also install these libraries directly from github to get the latest versions (**recommended**):
+```
+haxelib git hlopenal https://github.com/HaxeFoundation/hashlink master libs/openal
+haxelib git hlsdl https://github.com/HaxeFoundation/hashlink master libs/sdl
+haxelib git hldx https://github.com/HaxeFoundation/hashlink master libs/directx
+```
```

Comment:
Be sure to communicate that dependencies need to be installed from github and that this is the recommended way. Otherwise the reader will likely hit compilation errors by installing old library versions.